README.md

```diff
@@ -26,3 +26,164 @@ _If you get errors concerning the `FLASK_APP` environment not being set, try `$
 
 ### Database Schema
 ![image](https://user-images.githubusercontent.com/62635544/96819356-a6626380-13e0-11eb-8398-eef92ca100f3.png)
+
+### Endpoint Map
+
+### Applicants
+#### GET `/api/v1/applicants` 
+##### Response
+(all applicants)
+```
+{ 
+  data: [{
+​    "id": "1",
+​    "username": "Chipmunk",
+    "bio": "I'm the best one you could possibly hire",
+​    "updated_at": "Oct_20_etc_etc",
```

Comment:
so @gabichuelas and I are having a hard time figuring out how to include an `updated_at` attribute and are hoping it'd be alright if we plan to come back to this later when time allows, since the filtering by when a profile has been updated isn't exactly essential at this point? we'll still have a `created_at` attribute

Reply:
I'm fine with putting it on hold for now! @GreysonElkins @amykarnaze what do y'all think?

Reply:
I agree. We were thinking of using this as a part of our sorting for search results (not serving up accounts that have been inactive for 6 months above ones which were updated yesterday, for example) but this is not essential and can definitely happen sometime down the line. 

@ErinUntermeyer do we know if TypeScript will be mad if we leave our definitions as including `updated_at` when we start receiving data that doesn't have that property?

Reply:
TypeScript will be mad. We will have to match our definitions to the data we get back! But should be a very quick fix if the data changes in the future now that we have made a definitions file and only have to update it in a single place.

Reply:
All those things sound good!